cli/src/main.rs

```diff
@@ -447,4 +453,4 @@ fn setup_logger() -> anyhow::Result<()> {
     }
     builder.init();
     Ok(())
-}
+}
```

Comment:
I'm not sure why the Trunk Check runner isn't working, but it seems you need to run `trunk check -y` on this PR